sophia/src/error.rs

```diff
@@ -3,6 +3,9 @@
 use pest::error::{InputLocation, LineColLocation};
 
 error_chain! {
+    links {
+        XmlError(crate::parser::xml::error::Error, crate::parser::xml::error::ErrorKind);
```

Comment:
My idea was rather that all parsers would "raise" `ParserError`, possibly *chaining* their own specific error type.

Reply:
Ah yes, I see what you mean. Unfortunately `quick-xml` is only a streaming parser, so it does not store the line/col location, only the input location...

Reply:
Not a problem: you should use `LineColLocation::Pos(0,0)`.

I know, this should be documented :-/